src/main/webapp/EditHelp.js

```diff
@@ -0,0 +1,100 @@
+Behaviour.specify("A.help-button", 'hudson-behavior', 1, function(e) {
```

Comment:
@michaelneale @daniel-beck I commended several lines after Daniel's comment. Do you think more comments would be useful?